src/CacheManager.h

```diff
@@ -34,6 +34,9 @@ class CacheManager
 public:
     typedef std::vector<Mgr::ActionProfilePointer> Menu;
 
+    /// initial URL path characters that identify cache manager requests
+    static const SBuf &MagicUrlPathPrefix();
```

Comment:
IMO we should use the formal terminology here. Under IETF nomenclature these are called "well-known" prefixes. Even though Squid does not use the RFC 8615 "/.well-known/" string.

Reply:
Done in commit 4a3ec07.